### ISSUE TITLE
Simpsons, The - Bart vs. The Space Mutants (1991)(Ocean)[cr AGL].adf

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#638]
This game is broken in v1.1b8:

<img width="976" alt="Bildschirmfoto 2022-01-26 um 08 50 11" src="https://user-images.githubusercontent.com/12561945/151123639-4cfac418-4b85-46b6-b7b3-6a8e69bf7d95.png">

Worked fine in v1.1b7:

<img width="976" alt="Bildschirmfoto 2022-01-26 um 08 50 16" src="https://user-images.githubusercontent.com/12561945/151123712-88ec7974-ee39-4064-bbf0-c797b37514f6.png">

**Comment by emoon** [comment]
Hey,

I have a suggestion for this. While it will take a bit of work to d. I think it will be worth it in the long run.

As you have lots of tests in your test suite it would be a great idea to automate verifying that everything works as intended between releases. One way to do that would be to

1. Do a screen capture of the test in the emulator that you know is a good ref. Calculate a hash of this screenshot that is a "known good"
2. Make a very simple script in style of
3. start vamiga with adf/exe, wait n-number of frames. take screenshot, calculate hash. Verify against known good.
4. Run this as automation an GH to verify changes doesn't break anything.

**Comment by dirkwhoffmann** [comment]
> I have a suggestion for this

It's already there (at least partly).

During the last months I've added regression testing scripts for many tests. They look like this:

```
# Setup the test environment
regression setup A500_ECS_1MB /tmp/kick13.rom

# Run the test
regression run /tmp/bplptr6_ecs.adf
wait 9 seconds

# Exit with a screensht
screenshot set cutout 196 50 908 312
screenshot save bplptr6_ecs
```

There is also a Makefile that executes all existing script and compares the generated screenshots with a reference image.

The bug shows that my current test suite is not good enough.

**Comment by emoon** [comment]
Ah cool :)

**Comment by dirkwhoffmann** [comment]
Fixed in v1.1b9.
